lib/rules.rb

```diff
@@ -0,0 +1,88 @@
+require 'pry'
+
+class Rules
+  attr_reader :board
+
+  def initialize(board)
+    @board = board
+  end
+
+  def check_winner
```

Comment:
Renamed 'check_winner' to 'get_winning_mark' and the rest of win/tie methods to include 'is' prefix.

Reply:
Removed 'is' prefix from win/tie methods because they don't return booleans, they return the winning mark. Renamed to use 'get' prefix and 'mark' suffix (e.g. 'get_horizontal_winner_mark')